### ISSUE TITLE
Awesome theme doesn't generate .tex file

### ISSUE THREAD

**Issue by mtantawy** [hacksalot/HackMyResume#138]
HMR version 1.7.4

```
hackmyresume build resume.json to out/resume.all -t awesome -d

*** HackMyResume v1.7.4 ***
The -d or --debug switch was specified. DEBUG mode engaged.

  Platform:              linux
  Node.js:               v4.2.6
  HackMyResume:          v1.7.4
  FRESCA:                ~0.6.0

OPTIONS:

  theme                  awesome
  prettify               true
  css                    undefined
  pdf                    undefined
  sort                   true
  tips                   undefined
  debug                  true

Invoking BUILD command.
Reading FRESH resume: resume.json
Verifying AWESOME theme.
Applying AWESOME theme (3 formats)
Generating LATEX resume: out/resume.latex
Error: 
SyntaxError: Unexpected token ILLEGAL
    at Function (native)
    at Function._.template (/usr/lib/node_modules/hackmyresume/node_modules/underscore/underscore.js:1454:20)
    at Object.module.exports.generateSimple (/usr/lib/node_modules/hackmyresume/dist/renderers/underscore-generator.js:27:22)
    at Object.module.exports.generate (/usr/lib/node_modules/hackmyresume/dist/renderers/underscore-generator.js:56:19)
    at Class.module.exports.BaseGenerator.extend.single (/usr/lib/node_modules/hackmyresume/dist/generators/template-generator.js:155:20)
    at Class.<anonymous> (/usr/lib/node_modules/hackmyresume/dist/generators/template-generator.js:69:20)
    at Array.map (native)
    at Class.module.exports.BaseGenerator.extend.invoke (/usr/lib/node_modules/hackmyresume/dist/generators/template-generator.js:67:30)
    at Class.module.exports.BaseGenerator.extend.generate (/usr/lib/node_modules/hackmyresume/dist/generators/template-generator.js:95:22)
    at Class.single (/usr/lib/node_modules/hackmyresume/dist/verbs/build.js:251:29)
Generating JSON resume: out/resume.json
Generating YML  resume: out/resume.yml
```

**Comment by hacksalot** [comment]
Thanks for the report. This is a bit of a regression; should be fixed in today's 1.7.5 release.

**Comment by mtantawy** [comment]
Great!
Looking forward

**Comment by mtantawy** [comment]
Any updates about 1.7.5?

**Comment by hacksalot** [comment]
Minor delay @mtantawy -- integrating latest Awesome-CV updates from [upstream](https://github.com/posquit0/Awesome-CV). Above issue is resolved in [dev](https://github.com/hacksalot/HackMyResume/tree/dev).

1.7.5 should drop later today too.

**Comment by hacksalot** [comment]
Fixed in [1.8.0](https://github.com/hacksalot/HackMyResume/releases/tag/v1.8.0).
